fvtest/CMakeLists.txt

```diff
@@ -23,16 +23,10 @@ include(OmrAssert)
 
 omr_assert(TEST OMR_FVTEST)
 
-omr_assert(
-	TEST OMR_EXAMPLE
-	MESSAGE "The fv test component relies on the example glue"
-)
```

Comment:
This is the assert I'm concerned with removing. I don't think anyone has really thought about which tests need the example, and which don't.  gctest for sure needs the example, I suspect also rastest and vmtest. Maybe even sigtest.

Reply:
I've checked a bit: if there is a custom glue library, gctest can be built without examples, so I've removed the assert from fvtest (but I had an idea to move the assert to gctest's CMakeLists.txt directly).

Reply:
Moving the assert to gctest makes sense. The gctest definitely relies on the example's GC glue code.